src/components/popover/popover.component.scss

```diff
@@ -1,11 +1,66 @@
-:host {
+.popover {
     display: flex;
     flex-direction: column;
-    max-width: none;
     padding: 0;
+    max-width: 568px;
```

Comment:
And on that day I would like to still keep it the same, because otherwise popover would go out of the control and we would have a very big popover, where our goal is very compact popover.